### PR TITLE
Develop: Adding the Drop Down Menu for the languages in the "Add Snippet" page

### PR DESCRIPTION
**Pull request by Suhas-Koheda**
All the popular programming languages has been added in the drop down thereby removing the randomness of languages 

22 popular languages has been added 

1. Java
2. Python
3. JavaScript
4. C++
5. C#
6. Go
7. Kotlin
8. Ruby
9. Swift
10. PHP
11. TypeScript
12. Rust
13. Dart
14. Scala
15. Perl
16. R
17. Elixir
18. Haskell
19. Lua
20. C
21. MATLAB
22. Shell

### CODE REVIEW

**Comment by Suhas-Koheda** [comment]
@notsoocool hey i have given my pull request to the develop branch 
once check it, deploy it in the development server and then merge it with the main branch

File: .gitignore
Comment (notsoocool) [inline .gitignore:1]: Please Discard your changes here, no need to add .env to gitignore.

File: app/(dashboard)/addsnippets/page.tsx
Comment (notsoocool) [inline app/(dashboard)/addsnippets/page.tsx:1]: Perfect, It looks great, can you just decrease the speed of scroll up and down in the 'Dropdown Menu'.

**Comment by Suhas-Koheda** [comment]
i have removed .env from gitignore 
and the scroll wheel part i'll try to solve it using javascript
@notsoocool

**Comment by notsoocool** [comment]
just make the specific changes and i will label it as **_level2_**

**Comment by Suhas-Koheda** [comment]
@notsoocool check if its ok?
i tried  a lot

**Comment by notsoocool** [comment]
> @notsoocool check if its ok? i tried a lot actually what is level 2?

no worries i will change it myself, as it is your first pr for the repo don't pressurise yourself

levels are basically used by PA for gssoc to deduct your overall performance.

**Pull request merged**